tutorials/3-run-node/index.md

```diff
@@ -41,52 +41,135 @@ $ cd cosmos-sdk
 Make sure you are using the same version used at the time of writing:
 
 ```sh
-$ git checkout v0.42.6
+$ git checkout v0.45.4
 ```
 
-Then build `cosmos-sdk`:
+Then build `cosmos-sdk`. If you use Docker, with the help of a ready-made [multi-stage creation](https://github.com/cosmos/cosmos-sdk/blob/v0.45.4/Dockerfile#L37), you create a new Docker image that contains the compiled `simd`:
+
+<CodeGroup>
+
+<CodeGroupItem title="Local">
 
 ```sh
 $ make build
 ```
 
-The build takes a few minutes and creates a `build` folder and a `simapp` binary named `simd`:
+The build takes a few minutes and creates a `build` folder and a `simapp` binary named `simd`.
+
+</CodeGroupItem>
+
+<CodeGroupItem title="Docker">
 
 ```sh
-$ ls build
+$ docker build . -t simd:v0.45.4
+```
+
+</CodeGroupItem>
+
+</CodeGroup>
+
+Confirm that you got what you expected:
+
+<CodeGroup>
+
+<CodeGroupItem title="Local">
+
+```sh
+$ ./build/simd version
+```
+
+</CodeGroupItem>
+
+<CodeGroupItem title="Docker">
+
+```sh
+$ docker run --rm -it simd:v0.45.4 simd version
+```
+
+</CodeGroupItem>
+
+</CodeGroup>
+
+This should return:
+
+```txt
+0.45.4
 ```
 
 ## Initialize `simapp`
 
-Now reset the database. Run this step not only when the database has already been initialized but even if this is the first time you are testing `simapp`:
+Now reset the database. To help you ring-fence this exercise, you can use a [Git-ignored](https://github.com/cosmos/cosmos-sdk/blob/v0.45.4/.gitignore#L12) subfolder of the repository: `private`.
+
+Run this step not only when the database has already been initialized but even if this is the first time you are testing `simapp`.:
+
+<CodeGroup>
+
+<CodeGroupItem title="Local">
 
 ```sh
-$ cd build
-$ ./simd unsafe-reset-all
+$ ./build/simd tendermint unsafe-reset-all \
+    --home ./private/.simapp
```

Comment:
Just a heads up, I get a `dir=private/.simapp/data` in the output. Not sure if it is just an output error, perhaps it is better to go with `--home $(pwd)/private/.simapp` like you did in the next command.

Reply:
I noticed too that the `unsafe-reset-all` adds garbage not in `config`... I am thinking to either say nothing or add a small paragraph warning not to worry about the garbage.

Reply:
And `unsafe-reset-all` does not remove the `genesis.json`, which can cause some problems with the following commands if someone tries it again. Perhaps just let them know where the folder is and one can manually remove everything to have a real `reset-all`..

Reply:
Yeah, perhaps better to say delete this folder, all the more so that we use `./private`, right?

Reply:
Yes!